src/utils/findSectionForPath.js

```diff
@@ -16,7 +16,7 @@ import slugify from './slugify';
  * This method specifically works with the nav_*.yml format.
  */
 const findSectionForPath = (pathname, sections) => {
-  let activeSection;
+  let activeSection = sections[0];
```

Comment:
This behavior could cause false positives for pages that are, for example, in the `docs` folder but are intentionally no longer linked to from the side nav.